samples/costmanagement/README.md

```diff
@@ -0,0 +1,87 @@
+---
+page_type: sample
+languages:
+- python
+products:
+- azure
+description: "These code samples will show you how to manage Consumption using Azure SDK for Python."
+urlFragment: costmanagement
+---
+
+# Getting started - Managing Consumption using Azure Python SDK
+
+These code samples will show you how to manage costmanagement using Azure SDK for Python.
+
+## Features
+
+This project framework provides examples for the following services:
+
+### costmanagement
+* [] Using the Azure SDK for Python - costmanagement Management Library [azure-mgmt-costmanagement](https://pypi.org/project/azure-mgmt-costmanagement/) for the [costmanagement API](https://docs.microsoft.com/en-us/rest/api/costmanagement/)
```

Comment:
https://docs.microsoft.com/en-us/rest/api/costmanagement/ is broken link, please find a valid link